lib/std/testing.zig

```diff
@@ -539,7 +539,10 @@ fn printIndicatorLine(source: []const u8, indicator_index: usize) void {
         while (i < indicator_index) : (i += 1)
             print(" ", .{});
     }
-    print("^\n", .{});
+    if (indicator_index >= source.len)
+        print("^ (end of string)\n", .{})
+    else
+        print("^ ({d})\n", .{source[indicator_index]});
```

Comment:
Might be nice to format it as a character literal:

```suggestion
        print("^ ('\\x{x:0>2}')\n", .{source[indicator_index]});
```

This would give something like:

```
^ ('\x00')
```

Reply:
Alright, yeah, maybe that can help if the user wants to play around with it or something. I will use `\u{...}` so that it works with any character.

Reply:
Nevermind, it doesn't handle Unicode characters. Changed it to your original suggestion!